src/elements/content-sidebar/activity-feed/common/delete-confirmation/DeleteConfirmation.js

```diff
@@ -50,15 +51,26 @@ class DeleteConfirmation extends React.Component<Props> {
                 role="dialog"
                 shouldDefaultFocus
                 shouldOutlineFocus={false}
+                {...rest}
             >
                 <div className="bcs-DeleteConfirmation-promptMessage">
                     <FormattedMessage {...message} />
                 </div>
                 <div>
-                    <Button className="bcs-DeleteConfirmation-cancel" onClick={onDeleteCancel} type="button">
+                    <Button
+                        className="bcs-DeleteConfirmation-cancel"
+                        onClick={onDeleteCancel}
+                        type="button"
+                        data-resin-target={ACTIVITY_TARGETS.INLINE_DELETE_CANCEL}
+                    >
                         <FormattedMessage {...commonMessages.cancel} />
                     </Button>
-                    <PrimaryButton className="bcs-DeleteConfirmation-delete" onClick={onDeleteConfirm} type="button">
+                    <PrimaryButton
+                        className="bcs-DeleteConfirmation-delete"
+                        onClick={onDeleteConfirm}
+                        type="button"
+                        data-resin-target={ACTIVITY_TARGETS.INLINE_DELETE_CONFIRM}
```

Comment:
Since this is shared across tasks and comments, events can be distinguished by adding the `data-resin-component` prop to the component (it gets passed through the Overlay and FocusTrap).